src/uct/ib/mlx5/rc/rc_mlx5.inl

```diff
@@ -874,15 +881,46 @@ void uct_rc_mlx5_txqp_dptr_post_iov(uct_rc_mlx5_iface_common_t *iface, int qp_ty
         wqe_size         = ctrl_av_size + sizeof(*raddr) +
                            uct_ib_mlx5_set_data_seg_iov(txwq, (void*)(raddr + 1),
                                                         iov, iovcnt);
+        opmod            = 0;
         break;
 
+#if HAVE_MLX5_MMO
+    case MLX5_OPCODE_MMO|UCT_RC_MLX5_OPCODE_FLAG_MMO_PUT:
+    case MLX5_OPCODE_MMO|UCT_RC_MLX5_OPCODE_FLAG_MMO_GET:
+        gga_wqe                  = txwq->curr;
+#if ENABLE_DEBUG_DATA
+        gga_wqe->padding         = 0xdeadbeef;
+#endif
+        gga_wqe->be_opaque_lkey  = htobe32(opaque_mr->lkey);
+        gga_wqe->be_opaque_vaddr = htobe64((uintptr_t)opaque_mr->addr);
```

Comment:
minor: maybe we can precompute the byte order switch during mem reg or rkey unpack?